app/assets/javascripts/short-form/templates/b2-contact.html.slim

```diff
@@ -4,16 +4,16 @@
       | {{'B2_CONTACT.TITLE' | translate:applicantFirstName()}}
 
   .app-inner.inset.border-bottom
-    .form-group.has-option ng-class="{ error: $scope.requiredContactInformationMissing() }"
+    .form-group.has-option ng-class="{ error: inputInvalid('phone') }"
```

Comment:
how were you able to trigger inputInvalid for phone without calling the function to see if there was missing required contact info?